bin/opensfm_run_all

```diff
@@ -10,3 +10,6 @@ $DIR/opensfm match_features $1
 $DIR/opensfm create_tracks $1
 $DIR/opensfm reconstruct $1
 $DIR/opensfm mesh $1
+$DIR/opensfm undistort $1
+$DIR/opensfm compute_depthmaps $1
+$DIR/opensfm export_openmvs $1
```

Comment:
do we always want to export to `openmvs`?

Reply:
Good point - no we don't.  But it is helpful at least to always generate the point cloud.

Reply:
Yes, it makes sense to have all the steps to get the point cloud here so they are visible. Thanks for adding this. If you can remove the last line, i'll merge.